app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md

```diff
@@ -36,142 +38,106 @@ inset_text:
     title: Start your training this September
```

Comment:
@plisovin this inset text isn't showing up in my preview app but I'm not sure why - do you mind taking a look when you can?

Reply:
@EllieNodder are you talking about this text?

<img width="1057" alt="Screenshot 2023-07-04 at 12 19 20" src="https://github.com/DFE-Digital/get-into-teaching-app/assets/77098906/f117f14b-f520-4e4c-9ffd-e20edb20e947">

Reply:
no, this text: 
![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/94541493/15c300e4-f620-4423-b787-2856b4962b4d)